site/static/examples/kind-with-registry.sh

```diff
@@ -16,7 +16,7 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 containerdConfigPatches:
 - |-
-  [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:${reg_port}"]
+  [plugins."io.containerd.grpc.v1.cri".registry.mirrors."${reg_name}:5000"]
```

Comment:
this change is wrong

Reply:
this configuration as-is allows us to remap pulls for images at `localhost:$registry_port` to the registry container, for a consistent reference between host and nodes

you can remap anything to anything, this is remapping is performed in the image resolution code in containerd.

Reply:
so containerd sees `localhost:5001/foo` and instead it pulls from the endpoint we specify here.